src/xpk/core/core.py

```diff
@@ -1500,7 +1500,7 @@ def run_gke_node_pool_create_command(
           f' --region={zone_to_region(args.zone)}'
           ' --num-nodes=1'
           f' --machine-type={args.pathways_gce_machine_type}'
-          ' --scopes=storage-full,gke-default'
+          ' --scopes=storage-full,gke-default,"https://www.googleapis.com/auth/cloud-platform"'
```

Comment:
Please use cloud-platform instead of url

Reply:
We currently use the full cloud-platform link in other parts of the same file. Should we keep it consistent or replace it (need to verify `cloud-platform` on its own also works).

E.g. 
https://github.com/AI-Hypercomputer/xpk/blob/main/src/xpk/core/core.py#L1449

and

https://github.com/AI-Hypercomputer/xpk/blob/main/src/xpk/core/core.py#L1460

Reply:
Done. Replaced with constant URL instead. Thanks!